mmv1/products/dataplex/Datascan.yaml

```diff
@@ -79,6 +79,13 @@ examples:
     test_env_vars:
       project_name: 'PROJECT_NAME'
     exclude_docs: true
+  - name: 'dataplex_datascan_onetime_profile'
```

Comment:
Is the significance for this amount of new examples for testing or for user documentation? We have a lot of running tests, so keeping test bloat to a minimum is a maintenance goal. 

If we can capture the significant cases/behavior of the field in fewer examples, but want to keep some of the examples for user-facing documentation, then we can add `exclude_test: true` to some